pkg/services/ngalert/api/tooling/definitions/provisioning_policies.go

```diff
@@ -19,6 +19,16 @@ package definitions
 //       202: Ack
 //       400: ValidationError
 
+// swagger:route DELETE /api/v1/provisioning/policies provisioning stable RouteResetPolicyTree
```

Comment:
I debated between:
REST-style `DELETE .../policies`
and
RPC-style `POST .../policies/reset`

I went with delete because the semantics of the route match this a bit closer. This operation is fully idempotent. Although the resource is never fully deleted, it's as if the server replaces it with a default in the background, so it felt close enough to a REST-style DELETE.

Please let me know if you disagree about the RESTfulness of this operation. We can always go with the RPC-style representation if needed.